src/python/pants/option/errors.py

```diff
@@ -33,6 +33,17 @@ class BooleanOptionNameWithNo(RegistrationError):
     """Boolean option names cannot start with --no."""
 
 
+class DefaultValueType(RegistrationError):
+    """Default value {value_type}({default_value!r}) does not match option type {option_type}."""
+
+
+class DefaultMemberValueType(DefaultValueType):
+    """Default member value type missmatch.
+
+    Member value {value_type}({member_value!r}) does not match list option type {member_type}.
```

Comment:
I had to break this up, as a single line sentence, docformatter made a change to keep the line width down, and on the second run, would add another empty line to separate the heading from the "body", leaving just `{member_type}.` on a lone line. Not pretty.

So, perhaps the doc string for `DefaultValueType` ought to be rephrased as well, for symmetry.

Reply:
Hm, this feels like an antipattern that we're using docstrings for the error message. We used to do that with `./pants help`, and changed it to use a class property `help` because it was too hard to get formatting right.